docs/gen-ai/gen-ai-metrics.md

```diff
@@ -0,0 +1,157 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Generative AI metrics
+--->
+
+# Semantic Conventions for Generative AI Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+The conventions described in this section are specific to Generative AI
+applications.
+
+**Disclaimer:** These are initial Generative AI client metric instruments
+and attributes but more may be added in the future.
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Generative AI Operations](#generative-ai-operations)
+  - [Metric: `gen_ai.token.usage`](#metric-gen_aitokenusage)
+  - [Metric: `gen_ai.operation.duration`](#metric-gen_aioperationduration)
+
+<!-- tocstop -->
+
+## Generative AI Operations
+
+The following metric instruments describe Generative AI operations. An
+operation may be a request to an LLM, a function call, or some other
+distinct action within a larger Generative AI workflow.
+
+### Metric: `gen_ai.token.usage`
+
+This metric is [required][MetricRequired] when an operation involves the usage
+of tokens.
+
```

Comment:
We should specify the histogram boundaries. Not sure what should it be, but I hope 67108864 (2^14) "should be enough for everyone" 

```suggestion
This metric SHOULD be specified with [ExplicitBucketBoundaries](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.31.0/specification/metrics/api.md#instrument-advisory-parameters) of [1, 4, 16, 64, 256, 1024, 4096, 16384, 65536, 262144, 1048576, 4194304, 16777216, 67108864].
```